### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by juancruzsx**
# Summary
* Maven archetype implemented
* .gitignore added with basic files for Java project and other files for MacOS, Maven and IDEs

# Jira Card
* https://wolox-support.atlassian.net/browse/QAT-821

### CODE REVIEW

File: src/test/java/examples/users/users.feature [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    Then status 201
+
+    * def id = response.id
+    * print 'created id is: ', id
+
+    Given path id
+    # When method get
+    # Then status 200
+    # And match response contains user
+
```
Comment (borbonandres) [inline src/test/java/examples/users/users.feature]: Why this is in your code?, this is a example from other repo
Please, deleted! :)

File: src/test/java/karate-config.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  }
+  var config = {
+    env: env,
+	myVarName: 'someValue'
+  }
+  if (env == 'dev') {
+    // customize
+    // e.g. config.foo = 'bar';
+  } else if (env == 'e2e') {
+    // customize
```
Comment (borbonandres) [inline src/test/java/karate-config.js:15]: Can we will change to other environment and not e2e?, What other environment use to wolox?

Comment (juancruzsx) [reply src/test/java/karate-config.js:15]: I made some changes in this file on #4 (altough its more like a template).
This PR was focused on just setting the folder structure for the maven project and some basic files (this is the default karate-config for the generated archetype) so I didn't make any changes at that time

File: src/test/java/examples/users/users.feature (hunk 1 above)
Comment (juancruzsx) [reply src/test/java/examples/users/users.feature]: Yeah, I didn't delete the example folder to check the folder structure i'd be using but it shouldn't be here in future PR to masters (but as I made new branches from development to continue with the other features its still there on #2 , #3 and #4 ).
Anyway, yes, it shouldn't go to master. Its deleted now (and I should update the other branches / PR soon 😛)

Comment (borbonandres) [reply src/test/java/examples/users/users.feature]: Great! thks

File: src/test/java/karate-config.js (hunk 2 above)
Comment (borbonandres) [reply src/test/java/karate-config.js:15]: Yeah, I had seen yet, but i had post the comment yet :)
Thks for aclaration

**Review by borbonandres — APPROVED** [review]
PR Ok. The comments will be change in other PR

**Pull request merged**